oauth2client/contrib/gce.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, scope, **kwargs):
             scope: string or iterable of strings, scope(s) of the credentials
                    being requested.
         """
+        # if scope:
```

Comment:
Let's do something real, though - how about logging a warning, or better yet making use of the [warnings module](https://docs.python.org/2/library/warnings.html)?